tasks/main.yml

```diff
@@ -322,7 +322,7 @@
 
     - name: Set the 3979 traceflag
       command: >-
-        /opt/mssql/bin/mssql-conf traceflag 3979 "on"
+        /opt/mssql/bin/mssql-conf traceflag 3979 'on'
```

Comment:
I don't think you need quotes here at all

Reply:
true, fixed